airflow/models/pool.py

```diff
@@ -33,20 +32,19 @@ class Pool(Base):
     slots = Column(Integer, default=0)
     description = Column(Text)
 
-    default_pool_name = 'not_pooled'
+    default_pool_name = 'default_pool'
```

Comment:
```suggestion
    DEFAULT_POOL_NAME = 'default_pool'
```

to indicate this is a constant.